tests/integration/apps/fizz_buzz.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+[0..100].each do |i|
```

Comment:
I think you meant to add parens around this range.

```suggestion
(0..100).each do |i|
```